lib/jobs/dell-racadm-tool-job.js

```diff
@@ -41,32 +41,32 @@ function racadmSetBIOSJobFactory(
      * @param {String} taskId
      * @constructor
      */
-    function RacadmSetBiosJob(options, context, taskId) {
-        RacadmSetBiosJob.super_.call(this, logger, options, context, taskId);
+    function racadmToolJob(options, context, taskId) {
+        racadmToolJob.super_.call(this, logger, options, context, taskId);
```

Comment:
Change racadmToolJob to RacadmToolJob. Constructor should starts with upper case.